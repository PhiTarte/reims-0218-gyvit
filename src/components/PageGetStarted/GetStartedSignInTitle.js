//TODO CREATE SIGN IN TITLE
import React from "react"

const GetStartedSignInTitle = () => (
  <div>
    <p
      style={{
        height: "24px",
        "font-size": "24px",
        "font-weight": "normal",
        "font-style": "normal",
        "font-stretch": "normal",
        "line-height": "normal",
        "letter-spacing": "normal",
        "text-align": "left",
        color: "#3b425d"
      }}
    >
      Get started absolutely free
    </p>
    <p
      style={{
        width: "300px",
        height: "18px",
        fontSize: "18px",
        fontWeight: "normal",
        fontStyle: "normal",
        "letter-spacing": "normal",
        "text-align": "left",
        color: "#9da5c9"
      }}
    >
      Free forever. No credit card needed.
    </p>
  </div>
)

export default GetStartedSignInTitle
