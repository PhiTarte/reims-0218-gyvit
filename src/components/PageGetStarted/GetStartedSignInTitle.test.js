import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import GetStartedSignInTitle from "./GetStartedSignInTitle"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<GetStartedSignInTitle />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<GetStartedSignInTitle />).toJSON()
  expect(tree).toMatchSnapshot()
})
