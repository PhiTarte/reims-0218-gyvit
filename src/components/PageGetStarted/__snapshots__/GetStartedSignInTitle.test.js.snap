// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div>
  <p
    style={
      Object {
        "color": "#3b425d",
        "font-size": "24px",
        "font-stretch": "normal",
        "font-style": "normal",
        "font-weight": "normal",
        "height": "24px",
        "letter-spacing": "normal",
        "line-height": "normal",
        "text-align": "left",
      }
    }
  >
    Get started absolutely free
  </p>
  <p
    style={
      Object {
        "color": "#9da5c9",
        "fontSize": "18px",
        "fontStyle": "normal",
        "fontWeight": "normal",
        "height": "18px",
        "letter-spacing": "normal",
        "text-align": "left",
        "width": "300px",
      }
    }
  >
    Free forever. No credit card needed.
  </p>
</div>
`;
